interfaces/acados_template/acados_template/acados_ocp.py

```diff
@@ -217,196 +229,210 @@ def nx(self, nx):
         if type(nx) == int and nx > 0:
             self.__nx = nx
         else:
-            raise Exception('Invalid nx value. Exiting.')
+            raise Exception('Invalid nx value, expected integer. Exiting.')
```

Comment:
very minor remark, but the exception message does not match exactly the reason why it was raised. It could say `expected positive integer` or `non negative` for some others below

Reply:
right!
addressed, even though the idea is that these setters dont really have to be called anymore :)